pkg/providers/vsphere/vmprovider_vm_test.go

```diff
@@ -5052,3 +5064,26 @@ func getDVPG(
 
 	return network, dvpg
 }
+
+func updateVMBackupYAMLWithAPIVersion(
+	ctx *builder.TestContextForVCSim,
+	vm *vmopv1.VirtualMachine,
+	apiVersion string,
+	vcVM *object.VirtualMachine) {
+
+	vm.APIVersion = apiVersion
```

Comment:
Would it be better to set the APIVersion in the dummy VM builder?

Reply:
Cool, let me add a dummy func